### DISCUSSION TITLE
Sec 53.3.1 Tokio (dependencies)

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1315] [General]
I tried running this locally and it wasn't as easy as I expected. The bottom line is I had to add:

```
[dependencies]
tokio = { version = "1.32.0", features = ["full"] }
```

to my `Cargo.toml`. I expected to have to add `tokio`, but the `features=["full"]` was a surprise.

I feel as if this would be worth a note. (I wonder if you cover it later, but I'm trying things as I go, and this took some head-scratching and a few frustrating dead ends.)

**Comment by mgeisler** [comment]
@sakex, do you remember what the reference to a "prepared `main.rs`" means on [Join](https://google.github.io/comprehensive-rust/async/control-flow/join.html)?

**Comment by djmitche** [comment]
I think this was covered in another discussion (or two or three..)

It refers back to the class preparation.

**Reply by djmitche** [reply]
Ah, #1330.
